src/Sections/AdminLogin/index.module.scss

```diff
@@ -0,0 +1,95 @@
+.admin-login--container {
+	min-width: 100vw;
+	min-height: 100vh;
+	display: grid;
+	grid-template-columns: 50vw 50vw;
+}
+
+.admin-login--form-container {
+	min-height: 100vh;
+	background-color: #ffffff;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.admin-login--form-container > div {
```

Comment:
Why using > when you are using scss. Why don't you nested inside

Reply:
lol
i forgot